src/front/spv/mod.rs

```diff
@@ -1307,10 +1345,38 @@ impl<I: Iterator<Item = u32>> Parser<I> {
                     inst.expect(4)?;
                     self.parse_expr_unary_op(expressions, crate::UnaryOperator::Not)?;
                 }
+                Op::LogicalOr => {
+                    inst.expect(5)?;
+                    self.parse_expr_binary_op(expressions, crate::BinaryOperator::LogicalOr)?;
+                }
+                Op::LogicalAnd => {
+                    inst.expect(5)?;
+                    self.parse_expr_binary_op(expressions, crate::BinaryOperator::LogicalAnd)?;
+                }
                 op if inst.op >= Op::IEqual && inst.op <= Op::FUnordGreaterThanEqual => {
                     inst.expect(5)?;
                     self.parse_expr_binary_op(expressions, map_binary_operator(op)?)?;
                 }
+                op if inst.op >= Op::Any && inst.op <= Op::IsNormal => {
```

Comment:
I am not sure if we want to construct our matchings like this? It is okay for now, but it requires the one that is working with this code to have knowledge about what OP codes are defined between `Op::Any` and `Op::IsNormal`.

Reply:
It's not super neat, I agree. It's just there to save the code space, and it was already used right above, so nothing new this PR brings on the table. Do you think we should spell out each of the cases individually? That would be about a page long, but I think we could live with that.